pkg/template/helm_template.go

```diff
@@ -74,12 +84,18 @@ func (t *HelmTemplate) TemplateDir(dirPath string) (exec.CmdRunResult, bool) {
 
 		paths, valuesCleanUpFunc, err := vals.AsPaths(dirPath)
 		if err != nil {
-			return exec.NewCmdRunResultWithErr(err), true
+			return exec.NewCmdRunResultWithErr(err)
 		}
 
 		defer valuesCleanUpFunc()
 
 		for _, path := range paths {
+			if path == stdinPath {
+				if input == nil {
```

Comment:
can these be folded into `if path == stdinPath && input == nil` ?

Reply:
fixed